commands/hubspot.rb

```diff
@@ -0,0 +1,38 @@
+require_relative '../base_command'
+require "mechanize"
+require 'nokogiri'
+require 'open-uri'
+require 'pry'
+class Hubspot < BaseCommand
+  def perform
+    agent = Mechanize.new
+    page = agent.get('https://app.fooda.com/login')
+
+    # login
+    form = agent.page.forms.first
+    form.fields[2].value = 'food@food.com'
+    form.fields[3].value = 'Pursu1ts'
```

Comment:
DZ - It won't be more convenient when someone logins here and changes your password and you don't know what it was changed to.

<img width="1216" alt="screenshot 2016-02-26 19 38 36" src="https://cloud.githubusercontent.com/assets/1051633/13369252/932e4290-dcc0-11e5-8055-2141897f9512.png">

You can share the credentials in each `sample.env` or let each developer create their own test account that won't collide with the "real" Intrepid account.

It's also Factor 3 in having a 12 Factor App http://12factor.net/config which is a best practice.